xbmc/guilib/WindowIDs.h

```diff
@@ -82,6 +82,7 @@
 #define WINDOW_DIALOG_PROFILE_SETTINGS    10130
 #define WINDOW_DIALOG_LOCK_SETTINGS       10131
 #define WINDOW_DIALOG_CONTENT_SETTINGS    10132
+#define WINDOW_DIALOG_LIBEXPORT_SETTINGS  10133
```

Comment:
Has not been used since before Dharma, so whenever it was used it was a long time ago and not in Git. Therefore I think this is fine but I will change if required.

Reply:
nah. then it's fine. just worried about random leftover xml files in skins, but dharma is more than sufficiently back in time.